workflows/pkg/apis/workflows/v1alpha1/workflow_types.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
im curious what nonNamespaced is doing, looks like we don't have that for pipelines, also we have this "krshapedlogic=false" comment as well https://github.com/tektoncd/pipeline/blob/caa619b8d14999e8599f7ea86a7521e421d20c3e/pkg/apis/pipeline/v1beta1/pipelinerun_types.go#L34-L36 (maybe those are out of date tho)

Reply:
hmmm this looks like a bug -- most likely I copied it the genclient directives from a Cluster scoped resource instead of a Namespaced resource. Good catch

Reply:
looks like I had 2 genclients but non genreconciler